Sources/Classes/RSEventRepository.m

```diff
@@ -661,17 +661,24 @@ - (void)_applicationDidFinishLaunchingWithOptions:(NSDictionary *)launchOptions
         [applicationOpenedProperties setObject:url forKey:@"url"];
     }
 #endif
+    
+    // Session Tracking
+    // Automatic tracking session started
+    if (self->config.automaticSessionTracking) {
+        [[RSClient sharedInstance] startSession:[NSString stringWithFormat:@"%ld", [RSUtils getTimeStampLong]]];
+    }
+    
     [[RSClient sharedInstance] track:@"Application Opened" properties:applicationOpenedProperties];
     
 }
 
 - (void)_applicationWillEnterForeground {
-#if TARGET_OS_WATCH
+//#if TARGET_OS_WATCH
     if(self->firstForeGround) {
         self->firstForeGround = NO;
         return;
     }
-#endif
+//#endif
```

Comment:
After commenting the `#if TARGET_OS_WATCH` and `#endif` is causing issue with the Application Opened event. Can we please revert back this change.